.env.development

```diff
@@ -11,7 +11,7 @@ NEXT_PUBLIC_CUSTOM_RPCS={ "59140": ["https://consensys-zkevm-goerli-prealpha.inf
 NEXT_PUBLIC_DKEY=AmE4rjRHH0aLksP_-Nym-DCL_5qOHBYR77m7svbGyHm5
 NEXT_PUBLIC_ONRAMPER_API_KEY=pk_prod_01GWH1ZXS9JS04JJQ5W4KWQBFF
 NEXT_PUBLIC_ONRAMPER_ENABLED=true
-NEXT_PUBLIC_BACKEND_URL=https://api-develop.jumper.exchange/v1
+NEXT_PUBLIC_BACKEND_URL=https://api.jumper.exchange/v1
```

Comment:
Do not update those env variables please

Reply:
changed back.